browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -451,6 +461,8 @@ void BraveToolbarView::Layout(PassKey) {
   }
 
   if (!location_bar_is_wide_.GetValue()) {
```

Comment:
I'm not sure if it matters (maybe it's intentional), but I seem to always drop into this if statement for every key press. It seems odd that we need to keep resetting the bounds on every key press. Not sure if that's expected or possibly a hint about this behavior.

Reply:
It looks like `OnChange` upstream calls `InvalidateLayout` so I guess this is intentional

Reply:
OK, makes sense. Just seemed odd at first!